lib/menu.rb

```diff
@@ -0,0 +1,35 @@
+
+
+class Menu
```

Comment:
I think you were supposed to add a hash of items from a menu and display them (user story #1: I would like to see a list of dishes with prices) so maybe if you added that, it could really help with your testing!